### ISSUE TITLE
Cannot set property 'enabled' of undefined

### ISSUE THREAD

**Issue by josemarluedke** [rondale-sc/ember-cli-rails-addon#6]
Hello there. I started getting this error when I upgraded to v0.0.7. It still happening on v0.0.8.

```
BuildingBuilding...
Cannot set property 'enabled' of undefined
TypeError: Cannot set property 'enabled' of undefined
    at Class.module.exports.included (/Users/josemarluedke/Projects/Ruby/ember-cli-rails-heroku/app/frontend/node_modules/ember-cli-rails-addon/index.js:60:39)
    at EmberApp.<anonymous> (/Users/josemarluedke/Projects/Ruby/ember-cli-rails-heroku/app/frontend/node_modules/ember-cli/lib/broccoli/ember-app.js:217:15)
    at Array.filter (native)
    at EmberApp._notifyAddonIncluded (/Users/josemarluedke/Projects/Ruby/ember-cli-rails-heroku/app/frontend/node_modules/ember-cli/lib/broccoli/ember-app.js:212:45)
    at new EmberApp (/Users/josemarluedke/Projects/Ruby/ember-cli-rails-heroku/app/frontend/node_modules/ember-cli/lib/broccoli/ember-app.js:203:8)
    at Object.<anonymous> (/Users/josemarluedke/Projects/Ruby/ember-cli-rails-heroku/app/frontend/Brocfile.js:5:11)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
```

You can check the project on: https://github.com/josemarluedke/ember-cli-rails-heroku. What I'm working on with this project, is to find a solution to deploy it to production env using Heroku.

*Note:* This is happening on development environment.

Best, J.

**Comment by josemarluedke** [comment]
Found the problem. It looks like the `ember-cli-rails-addon` was loading before the `broccoli-asset-rev` where it defines the `app.options.fingerprint.enabled `.

**Comment by rondale-sc** [comment]
Cool.  Thanks!  Glad it worked out, :)

**Comment by hank-harvey** [comment]
@josemarluedke @rondale-sc What was the process for resolving this? I'm getting the same thing after upgrading to 1.13.5

**Comment by rondale-sc** [comment]
@josemarluedke I believe https://github.com/rondale-sc/ember-cli-rails-addon/pull/13 fixes this.  If you point to master you should be able to test it out.  I'm going to cut a release shortly (just need to talk to @rwz)

**Comment by nickrivadeneira** [comment]
This was fixed for me when I `ember-cli-rails` gem to 0.3.5 and then updated `ember-cli-rails-addon` to 0.0.12.
